Components/Atoms/GoalListItem.js

```diff
@@ -45,15 +45,20 @@ const GoalButtonFront = ({text,totalCount,pushGoal,lastTimeStampMessage,is_overd
 
     return (
         <TouchableWithoutFeedback  onPress={pushGoal}>
-                    <View style={styles.box}>
-            <View style={styles.row}>
+            <Neomorph 
```

Comment:
Within the documentation, it looks like they can set an inner property which would be useful if we want to animate the process of pressing the button (e.g. when the button is pressed, it looks like the button is going in). 
<img width="588" alt="Screenshot 2020-05-09 13 08 05" src="https://user-images.githubusercontent.com/34425825/81480210-2d869b00-91f6-11ea-932a-fb1ff8c4e8c9.png">

Reply:
We can save this idea I just mentioned for another pull request though, since its a bit more involved. I was just playing around with that inner property and got this error 
<img width="352" alt="Screenshot 2020-05-09 13 09 55" src="https://user-images.githubusercontent.com/34425825/81480264-7d656200-91f6-11ea-81ba-5baecc9187d4.png">